python/test/t_IndependentMetropolisHastings_std.py

```diff
@@ -2,18 +2,54 @@
 
 import openturns as ot
 import openturns.testing as ott
-from math import pi
+import math
 
 ot.TESTPREAMBLE()
 
+ot.RandomGenerator.SetSeed(0)
+
+# %%
+# Test independentMetropolisHastings on Beta-Binomial conjugate model
+
+# Define Beta-binomial model
+a, b, lower, upper = 1.0, 1.0, 0.0, 1.0
+n, p = 10, 0.5
+prior = ot.Beta(a, b, lower, upper)
+model = ot.Binomial(n, p)
+
+# %%
+# Simulate data and compute analytical posterior
+X = model.getSample(1)
```

Comment:
According to PEP8, variable names should have a lower case first letter (see [peps.python.org](https://peps.python.org/pep-0008/#function-and-variable-names)). This is why github colors this variable in orange, as if this was a class name.

![image](https://github.com/openturns/openturns/assets/31351465/a3364664-4527-4343-8581-761bdd6923b5)

```suggestion
x = model.getSample(1)
```